lib/redcord/vacuum_helper.rb

```diff
@@ -59,4 +72,16 @@ def self._remove_stale_ids_from_sorted_set(model, sorted_set_key)
       end
     end
   end
+
+  sig { params(model: T.class_of(Redcord::Base), hash_tag: String, index_name: Symbol).void }
+  def self._remove_stale_records_from_custom_index(model, hash_tag, index_name)
+    index_key = "#{model.model_key}:custom_index:#{index_name}#{hash_tag}"
+    index_content_key = "#{model.model_key}:custom_index:#{index_name}_content#{hash_tag}"
+    model.redis.hgetall(index_content_key).each do |id, index_string|
```

Comment:
I think hgetall here would be a potentially expensive operation

Reply:
Agree, changing to scan.